wayat/frontend/ios/Runner/AppDelegate.swift

```diff
@@ -1,13 +1,17 @@
 import UIKit
 import Flutter
+import GoogleMaps
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+    
   ) -> Bool {
+    GMSServices.provideAPIKey("API-KEY")
```

Comment:
Here you need to pass the API-KEY from the environment variables, it should be called GOOGLE_MAPS_KEY

Reply:
fix